pkg/operation/cloudbotanist/gcpbotanist/infrastructure.go

```diff
@@ -58,16 +58,17 @@ func (b *GCPBotanist) DestroyInfrastructure() error {
 		return err
 	}
 	var (
-		g                                  = flow.NewGraph("GCP infrastructure destruction")
-		destroyKubernetesFirewallRulesStep = g.Add(flow.Task{
-			Name: "Destroying Kubernetes firewall rules",
-			Fn:   flow.TaskFn(b.destroyKubernetesFirewallRules).RetryUntilTimeout(10*time.Second, 5*time.Minute).DoIf(configExists),
+		g = flow.NewGraph("GCP infrastructure destruction")
+
+		destroyKubernetesFirewallRulesAndRoutesStep = g.Add(flow.Task{
+			Name: "Destroying Kubernetes firewall rules and routes",
+			Fn:   flow.TaskFn(b.destroyKubernetesFirewallRulesAndRoutes).RetryUntilTimeout(10*time.Second, 5*time.Minute).DoIf(configExists),
```

Comment:
Why not making destroyRoutes a separate step in the flow?

Reply:
I also thought about that, but then did it similar to the aws implementation where the deletion of the loadbalancer and security group is summarized.

Reply:
Okay, fair enough.

I think both the firewall rules as well as the routes can be deleted in parallel, can't they? 
So, this would be a minor benefit of having to separate steps in the flow.

Reply:
That's a valid point, I've adapted it accordingly.